src/server/controllers/match.controller.js

```diff
@@ -80,37 +138,52 @@ function arrayContainsElement(arr, val) {
 
 function checkCareerFilter(userInfo, filter) {
```

Comment:
What does this function do? "check" doesn't tell me what it does - it seems like it's just checking for preconditions for the filter object, but not sure why these are required. Either change the method name or comment on what this function does?